Description: adjust tests to tidy-html5 5.6
 and the cherry-picked patches from upstream 5.7 in Debian's
 tidy-html5 2:5.6.0-7
Origin: vendor
Bug: https://bugs.debian.org/910943
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2018-11-15

--- a/t/cfg-for-parse.cfg
+++ b/t/cfg-for-parse.cfg
@@ -1 +1,2 @@
 show-warnings: 0
+show-info: 0
--- a/t/ignore-text.t
+++ b/t/ignore-text.t
@@ -21,6 +21,7 @@
     my $tidy = HTML::Tidy->new;
     isa_ok( $tidy, 'HTML::Tidy' );
 
+    $tidy->ignore( type => TIDY_INFO );
     $tidy->ignore( text => qr/bogotag/ );
     $tidy->ignore( text => [ qr/UNESCAPED/, qr/case-insensitive/i ] );
     # The qr/UNESCAPED/ should not ignore anything because there's no /i
--- a/t/extra-quote.t
+++ b/t/extra-quote.t
@@ -22,13 +22,14 @@
 my @expected = split /\n/, q{
 - (4:1) Warning: <img> unexpected or duplicate quote mark
 - (4:1) Warning: <img> escaping malformed URI reference
+- (4:1) Warning: <img> illegal characters found in URI
 - (4:1) Warning: <img> lacks "alt" attribute
 };
 chomp @expected;
 shift @expected; # First one's blank
 
 my @messages = $tidy->messages;
-is( scalar @messages, 3, 'Should have exactly three messages' );
+is( scalar @messages, 4, 'Should have exactly four messages' );
 
 my @strings = map { $_->as_string } @messages;
 s/[\r\n]+\z// for @strings;
--- a/t/ignore.t
+++ b/t/ignore.t
@@ -27,7 +27,7 @@
     my $tidy = HTML::Tidy->new;
     isa_ok( $tidy, 'HTML::Tidy' );
 
-    $tidy->ignore( type => TIDY_ERROR );
+    $tidy->ignore( type => TIDY_ERROR, type => TIDY_INFO );
     my $rc = $tidy->parse( '-', $html );
     ok( $rc, 'Parsed OK' );
 
@@ -41,7 +41,7 @@
     my $tidy = HTML::Tidy->new;
     isa_ok( $tidy, 'HTML::Tidy' );
 
-    $tidy->ignore( type => TIDY_WARNING );
+    $tidy->ignore( type => TIDY_WARNING, type => TIDY_INFO );
     my $rc = $tidy->parse( '-', $html );
     ok( $rc, 'Parsed OK' );
 
--- a/t/levels.t
+++ b/t/levels.t
@@ -9,6 +9,7 @@
 
 my $tidy = HTML::Tidy->new;
 isa_ok( $tidy, 'HTML::Tidy' );
+$tidy->ignore ( type => TIDY_INFO );
 my $rc = $tidy->parse( '-', <DATA> );
 ok( $rc, 'Parsed OK' );
 
--- a/t/simple.t
+++ b/t/simple.t
@@ -17,7 +17,7 @@
 ok( $rc, 'Parsed OK' );
 
 my @messages = $tidy->messages;
-is( scalar @messages, 5, 'Right number of initial messages' );
+is( scalar @messages, 6, 'Right number of initial messages' );
 
 $tidy->clear_messages;
 is_deeply( [$tidy->messages], [], 'Cleared the messages' );
--- a/t/too-many-titles.t
+++ b/t/too-many-titles.t
@@ -11,7 +11,7 @@
 
 my @expected = split /\n/, q{
 - (1:1) Warning: missing <!DOCTYPE> declaration
-- (4:9) Warning: too many title elements in <head>
+- (2:5) Warning: too many title elements in <title>
 };
 chomp @expected;
 shift @expected; # First one's blank
--- a/t/opt-00.t
+++ b/t/opt-00.t
@@ -42,7 +42,7 @@
 </title>
 <style type="text/css">
 /*<![CDATA[*/
- span.myprefix-1 {color: blue}
+ span.myprefix1 {color: blue}
 /*]]>*/
 </style>
 </head>
@@ -51,7 +51,7 @@
 
 <p>Here's some <b>ed and
 <br />
-eakfest MarkUp: <span class="myprefix-1">...</span></b></p>
+eakfest MarkUp: <span class="myprefix1">...</span></b></p>
 
 <p>
 </p>
